step/src/main/java/com/google/sps/servlets/PrefServlet.java

```diff
@@ -26,11 +30,31 @@
 
 @WebServlet("/Pref")
 public class PrefServlet extends HttpServlet {
-    // The class stores the user' preferences data
-    @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html");
-    PrintWriter out = response.getWriter();
-    out.println("Welcome to Friend Matching Plus");
+  // The class stores the user' personal information data
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String q1 = request.getParameter("q1");
+    String q2 = request.getParameter("q2");
+    String q3 = request.getParameter("q3");
+    String q4 = request.getParameter("q4");
+    String q5 = request.getParameter("q5");
```

Comment:
This makes sense but we might want to brainstorm how to make our form more flexible for the future so that we can add and remove questions without having to change too much infrastructure.

Reply:
I agree with that. I might see if there is anyway to organize it better